RUFAS/data_validator.py

```diff
@@ -179,8 +176,10 @@ def get_modifiable_at_runtime(cls) -> List["Modifiability"]:
 class DataValidator:
     """This class is will be utilized to validate all types of data across RuFas codebase."""
 
-    @staticmethod
-    def validate_properties(metadata: Dict[str, Any], metadata_depth_limit: int) -> Tuple[bool, str]:
+    def __init__(self) -> None:
+        self.event_logs: list[dict[str, str | dict[str, str]]] = []
+
+    def validate_properties(self, metadata: Dict[str, Any], metadata_depth_limit: int) -> Tuple[bool, str]:
```

Comment:
```suggestion
    def validate_properties(self, metadata: dict[str, Any], metadata_depth_limit: int) -> tuple[bool, str]:
```
I think it's preferred practice to use the native python lowercase type hints where possible (doesn't yet work with Any).